migrate/migrations/000007_add-posts-table.up.sql

```diff
@@ -0,0 +1,31 @@
+CREATE TABLE posts (
+    id INT UNSIGNED AUTO_INCREMENT PRIMARY KEY,
+    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+
+    type TINYINT UNSIGNED NOT NULL CHECK (type IN (0,1,2,3,4)),
+    audience TINYINT UNSIGNED NOT NULL CHECK (audience IN (0,1,2)),
+
+    author_id INT UNSIGNED NOT NULL,
+    title VARCHAR(256) NULL,
+    excerpt VARCHAR(500) NULL,
```

Comment:
This exceeds Ghost's custom_excerpt field so I think we're good

Reply:
Yep, i think Djordje mentioned on the frontend its expected not be greater than 400 chars max